### PR TITLE
field route added and also tested

### PR DESCRIPTION
**Pull request by Asakiz**

### CODE REVIEW

File: updatehub/src/settings.rs [hunk 1]
```diff
@@ -176,8 +177,12 @@ impl Default for Network {
         let server_address = mockito::server_url().to_string();
         #[cfg(not(test))]
         let server_address = "https://api.updatehub.io".to_string();
+        let route = "localhost:8080".to_string();
 
-        Self { server_address }
+        Self {
+            server_address,
+            route,
+        }
```
Comment (otavio) [inline updatehub/src/settings.rs]: It seems using the value without a var is clearer. We use the `server_address` was used as it varies on test builds.

```rust
impl Default for Network {
    fn default() -> Self {
        #[cfg(test)]
        let server_address = mockito::server_url().to_string();
        #[cfg(not(test))]
        let server_address = "https://api.updatehub.io".to_string();

        Self {
            server_address,
            route: "localhost:8080".to_string(),
        }
    }
}
```

File: updatehub/src/settings.rs [hunk 2]
```diff
@@ -168,6 +168,7 @@ impl Default for Update {
 #[serde(rename_all = "PascalCase")]
 pub struct Network {
     pub server_address: String,
+    pub route: String,
```
Comment (otavio) [inline updatehub/src/settings.rs]: The field is wrong, please check the reference code in Go.

File: updatehub/src/states/mod.rs [hunk 3]
```diff
@@ -214,7 +214,9 @@ impl StateMachine {
 /// # Ok(())
 /// # }
 /// ```
+
```
Comment (otavio) [inline updatehub/src/states/mod.rs]: This newline is wrong as it is documentation.

Comment (Jonathas-Conceicao) [inline updatehub/src/states/mod.rs]: This change is unnecessary.

File: updatehub/src/settings.rs [hunk 4]
```diff
@@ -233,6 +238,7 @@ MetadataPath=/tmp/metadata
         },
         network: Network {
             server_address: "http://localhost".into(),
+            listen_socket: "tcp://localhost:8081".into(),
```
Comment (otavio) [inline updatehub/src/settings.rs]: The `tcp://` prefix is a valid one for Rust?

Comment (Asakiz) [reply updatehub/src/settings.rs]: i checked and "tcp;//" is not a valid prefix, however the actix_web use a TCP in his implementation., therefore he guarentees the connection will be using TCP.

**Pull request merged**